apps/standalone/deploy/helm/flightctl-ui/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v2
 name: flightctl-ui
-description: A helm chart for flightctl-ui and keycloak
+description: A helm chart for flightctl UI
 type: application
 version: 0.0.1
```

Comment:
Should we maybe up the `version`? (Same for the rest of the OCP plugin chart)

Reply:
1. When we do the helm publish via an OCI container, it's injected into the Chart.yaml